### DISCUSSION TITLE
How to resolve AttributeError: module 'nicegui.events' has no attribute 'GenericEventArguments'?

### DISCUSSION THREAD

**Discussion by dontlagother** [zauberzeug/nicegui#2318] [Q&A] [answered]
### Question

When I ran the given code for Editable table, something went wrong
<img width="369" alt="2" src="https://github.com/zauberzeug/nicegui/assets/90087852/f3943c37-bdfc-48ec-bbc2-104bb14ce3dd">
<img width="452" alt="3" src="https://github.com/zauberzeug/nicegui/assets/90087852/e79e8a35-e7cb-4c7c-9c6f-f08e7dfc2319">
ps: my nicegui version is 1.2.24

**Comment by falkoschindler** [comment, marked as answer]
Hi @dontlagother, your NiceGUI version is pretty old. `GenericEventArguments` have been introduced in [1.3.0](https://github.com/zauberzeug/nicegui/releases/tag/v1.3.0).

**Reply by dontlagother** [reply]
Thank you for your answer!
